wallet-new/integration/Functions.hs

```diff
@@ -135,8 +161,11 @@ runAction wc action = do
                     <*> pure CreateWallet
 
         GetWallets -> do
-            -- our internal state is less important than
             result  <-  respToRes $ getWallets wc
+--            localWallets <- use wallets
+--            checkInvariant
+--                (length localWallets == length result)
+--                (LocalWalletsDiffers result localWallets)
```

Comment:
I'm commenting this out for the same reason that I've commented out a further problem. Trying to keep all of the details up to date in the local copy of the state is only going to cause massive complexity for questionable benefit. More deterministic unit tests can be used to verify more specific concerns, with a finer control on state.